src/index.js

```diff
@@ -43,11 +43,15 @@ async function nwbuild(options) {
 
   try {
     // Parse options
-    options = await util.parse(options, manifest);
```

Comment:
I remember why I did this. If options is empty, then getNodeManifest function will throw an error.

Reply:
@Black-Platypus ping